source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -1,181 +1,191 @@
-
 New Command-Based Library
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Watchdog and epoch reporting has been added to the command scheduler. This will let teams know exactly which command or subsystem is responsible for a loop overrun if one occurs.
-
-- Added a ``withName()`` command decorator for Java teams. This lets teams set the name of a particular command using the :ref:`decorator pattern <docs/software/commandbased/convenience-features:withName (Java only)>`.
-
-- Added a ``NetworkButton`` class, allowing users to use a boolean ``NetworkTableEntry`` as a button to trigger commands.
-
-- Added a ``simulationPeriodic()`` method to Subsystem. This method runs periodically during simulation, in addition to the regular ``periodic()`` method.
-
 General Library
 ^^^^^^^^^^^^^^^
 
-- Holonomic Drive Controller
-  - A controller that teams with holonomic drivetrains (i.e. swerve and mecanum) can use to follow trajectories. This also supports custom ``Rotation2d`` heading inputs that are separate from the trajectory because heading dynamics are decoupled from translational movement in holonomic drivetrains.
-
-- Added support for scheduling functions more often than the robot loop via ``addPeriodic()`` in TimedRobot. Previously, teams had to make a Notifier to run feedback controllers more often than the TimedRobot loop period of 20ms (running TimedRobot more often than this is not advised). Now, users can run feedback controllers more often than the main robot loop, but synchronously with the TimedRobot periodic functions so there aren't any thread safety issues. See an example :ref:`here <docs/software/convenience-features/scheduling-functions:Scheduling Functions at Custom Frequencies>`.
-
-- Added a ``toggle()`` function to Solenoid and DoubleSolenoid.
-
-- Added a ``SpeedControllerGroup`` constructor that takes a ``std::vector<>`` (C++) / ``SpeedController[]`` (Java), allowing the list to be constructed dynamically. (Teams shouldn't use this directly. This is only intended for bindings in languages like Python.)
 
-- Added methods (``isOperatorControlEnabled()`` and ``isAutonomousEnabled()``) to check game and enabled state together.
+- Rewrite Mechanism2d
+- Various ``getInstance()`` functions have been deprecated and replaced with static functions.
+- Add support for DMA to Java
+- ``Counter`` has been deprecated. Please use ``ExternalDirectionCounter``, ``Tach``, ``UpDownCounter``
+- ``Encoder`` has been deprecated. Please use ``QuadratureEncoder`` instead
+- Added C++ TankDrive example
+- GenericHID has been updated to static functions for non-defined controller types
```

Comment:
If this refers to wpilibsuite/allwpilib#3345 then this description isn't accurate. No static functions have been added... perhaps you mean that GenericHID is now concrete?

Also, I'd add something about replacing the `Hand` enum with side-specific functions.